src/test/java/net/ripe/rpki/rsyncit/rrdp/RrdpFetcherTest.java

```diff
@@ -15,6 +15,7 @@
 import java.time.Instant;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
 import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
Remove this one and replace by `assertThatThrownBy`?

Reply:
Good catch. Not sure why this is imported by default on my laptop.